### PR TITLE
Update facet field for data quality

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?
For https://github.com/sul-dlss/dor_indexing_app/pull/442

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -122,8 +122,7 @@ class CatalogController < ApplicationController
     add_common_date_facet_fields_to_config! config
 
     # This will help us find records that need to be fixed before we can move to cocina.
-    config.add_facet_field 'source_id_errors_ssim', label: 'Source ID Errors', home: false,
```
Comment (ndushay) [inline app/controllers/catalog_controller.rb:125]: I get the addition but why the removal of this facet?  It's not mentioned in the PR description or the ticket.

Comment (jcoyne) [reply app/controllers/catalog_controller.rb:125]: It was renamed because it now has more than source id errors.

Comment (ndushay) [reply app/controllers/catalog_controller.rb:125]: ah!  thank you.

**Review by ndushay — APPROVED** [review]
approve pending green for tests.

**Pull request merged**